test/Microsoft.AspNetCore.Blazor.Browser.Test/Interop/JavaScriptInvokeTests.cs

```diff
@@ -0,0 +1,429 @@
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.EchoFourParameters)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.EchoFiveParameters)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.EchoSixParameters)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.EchoSevenParameters)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.EchoEightParameters)));
+
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.VoidParameterlessAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.VoidWithOneParameterAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.VoidWithTwoParametersAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.VoidWithThreeParametersAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.VoidWithFourParametersAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.VoidWithFiveParametersAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.VoidWithSixParametersAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.VoidWithSevenParametersAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.VoidWithEightParametersAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.ReturnArrayAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.EchoOneParameterAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.EchoTwoParametersAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.EchoThreeParametersAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.EchoFourParametersAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.EchoFiveParametersAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.EchoSixParametersAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.EchoSevenParametersAsync)));
+                result.Add(CreateMethodOptions(nameof(JavaScriptInterop.EchoEightParametersAsync)));
+
+                return result;
+
+                MethodOptions CreateMethodOptions(string methodName) =>
+                    new MethodOptions
+                    {
+                        Type = new TypeInstance
+                        {
+                            Assembly = typeof(JavaScriptInterop).Assembly.GetName().Name,
+                            TypeName = typeof(JavaScriptInterop).FullName
+                        },
+                        Method = new MethodInstance
+                        {
+                            Name = nameof(JavaScriptInterop.VoidParameterless)
```

Comment:
I think the value here should be `methodName`. Otherwise it looks like it's repeating the same test case every time :)